.travis.yml

```diff
@@ -26,7 +26,7 @@ addons:
 before_install:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update ; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install gnu-tar dpkg libicns; fi
-  - gem install fpm
+  - gem install --no-rdoc fpm
```

Comment:
OS X Travis doesn't support cache and build is very slow — first attempt to speed up.